bundles/org.openhab.binding.evohome/src/main/java/org/openhab/binding/evohome/internal/handler/EvohomeAccountBridgeHandler.java

```diff
@@ -163,27 +184,32 @@ private boolean checkInstallationInfoHasDuplicateIds(Locations locations) {
         return result;
     }
 
+    @SuppressWarnings("null")
```

Comment:
I think this is hiding a problem, see below.

Reply:
Because the block is `synchronized` it should not be needed to use a local variabel. The compiler still shows a warning, i have seen this pattern in many bindings. I could change it to a local variabel. But why is that preferred over a synchronized block?

Reply:
Hmm, is it really needed to synchronize that block over just using a local variable?

Reply:
i guess not. changed it. thanks